.github/workflows/test.yml

```diff
@@ -17,6 +17,6 @@ on:
 jobs:
   test:
     if: github.repository == 'remix-run/remix'
-    uses: remix-run/remix/.github/workflows/reusable-test.yml@main
+    uses: jenseng/remix/.github/workflows/reusable-test.yml@fix-yarn-caching
```

Comment:
converted to a draft due to this testing chicken/egg problem ... temporarily setting this so the change actually gets tested :)

Reply:
tests passed using this workflow definition: https://github.com/remix-run/remix/actions/runs/2579571547, i've gone ahead and removed this tweak